src/main/java/by/exadel/internship/dto/Internship.java

```diff
@@ -0,0 +1,54 @@
+package by.exadel.internship.dto;
+
+
+import by.exadel.internship.dto.enums.FormatOfInternship;
+import by.exadel.internship.dto.enums.Technology;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
```

Comment:
the Constructor will have too much arguments, better to use @Builder or common  setters

Reply:
done fore DTOs that have a lot of arguments